### PR TITLE
Project cleanup after API/RI split.

### PR DESCRIPTION
**Pull request by Tomas-Kraus**

### CODE REVIEW

File: api/pom.xml [hunk 1]
```diff
@@ -90,12 +94,126 @@
...
+                </includes>
+            </resource>
+        </resources>
+
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.glassfish.hk2</groupId>
+                    <artifactId>osgiversion-maven-plugin</artifactId>
+                    <version>3.0.2</version>
```
Comment (lukasj) [inline api/pom.xml:121]: this plugin can go away, it is no longer used by the build

File: api/pom.xml [hunk 2]
```diff
@@ -15,20 +15,45 @@
...
+    <artifactId>jakarta.activation-api</artifactId>
     <version>2.1.0-SNAPSHOT</version>
-    <name>Jakarta Activation distribution</name>
-    <description>${project.name}</description>
+    <packaging>jar</packaging>
+    <name>Jakarta Activation API</name>
     <url>https://github.com/eclipse-ee4j/jaf</url>
 
+    <properties>
+        <spec.version>2.0</spec.version>
```
Comment (lukasj) [inline api/pom.xml:34]: should be 2.1

File: api/pom.xml [hunk 3]
```diff
@@ -15,20 +15,45 @@
...
+        <spotbugs.threshold>High</spotbugs.threshold>
+        <!--Maven plugins version-->
+        <spotbugs.plugin.version>4.2.0</spotbugs.plugin.version>
+
+        <findbugs.skip>
+            false
+        </findbugs.skip>
+        <findbugs.exclude>
+            ${project.basedir}/exclude.xml
+        </findbugs.exclude>
```
Comment (lukasj) [inline api/pom.xml:54]: I'd collapse the two props above to one line/property; no reason for one prop to occupy 3 lines

File: api/pom.xml [hunk 4]
```diff
@@ -217,27 +296,58 @@
...
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <configuration>
+                    <instructions>
+                        <Bundle-Description>${project.name} ${spec.version} Specification</Bundle-Description>
```
Comment (lukasj) [inline api/pom.xml:321]: rather define project.description and drop this line

File: api/pom.xml [hunk 5]
```diff
@@ -365,6 +475,7 @@ Use is subject to <a href="{@docRoot}/doc-files/speclicense.html" target="_top">
                     <docfilessubdirs>true</docfilessubdirs>
                     <javadocDirectory>${main.basedir}/activation/src/main/javadoc</javadocDirectory>
                     <detectJavaApiLink>false</detectJavaApiLink>
+                    <release>11</release>
```
Comment (lukasj) [inline api/pom.xml:478]: double check the `${current.year}` prop is resolved properly on generated javadoc as well as pointer to speclicense file works

Comment (Tomas-Kraus) [reply api/pom.xml:478]: This looks ok:
Comments to : jaf-dev@eclipse.org.
Copyright © 2019, 2021 Eclipse Foundation. All rights reserved.
Use is subject to license terms.
Locally I got file:///data/metro/jaf/api/target/apidocs/doc-files/speclicense.html URL for speclicense. The same pattern is being used in JAX-WS API so hope it's OK.

File: api/pom.xml (hunk 1 above)
Comment (Tomas-Kraus) [reply api/pom.xml:121]: Done.

File: api/pom.xml (hunk 2 above)
Comment (Tomas-Kraus) [reply api/pom.xml:34]: Done

File: api/pom.xml (hunk 3 above)
Comment (Tomas-Kraus) [reply api/pom.xml:54]: Done

File: api/pom.xml (hunk 4 above)
Comment (Tomas-Kraus) [reply api/pom.xml:321]: Done

**Review by lukasj — APPROVED** [review]
LGTM

**Pull request merged**